keyboards/xd75/keymaps/cryptokid91/keymap.c

```diff
@@ -0,0 +1,177 @@
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include QMK_KEYBOARD_H
+
+// Custom keycodes for longer things
+#define MS_LW LT(_LOWER, KC_BSPC)
+#define MS_RS LT(_RAISE, KC_DEL)
+#define MS_US MT(MOD_RSFT, KC_UP)
+
+// Layer shorthand
+#define _QWERTY 0
+#define _LOWER  1
+#define _RAISE  2
+#define _RESET  3
+
+enum custom_keycodes {
+  MS_P00 = SAFE_RANGE,
+  NEWTAB,
+  ROLL,
+  IOTECH,
+	TERM
+};
+
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+
+/* SPLIT QWERTY
+ * .--------------------------------------------------------------------------------------------------------------------------------------.
+ * | ESC    | 1      | 2      | 3      | 4      | 5      | PRTSCR | SLCK   | LOCK   | 6      | 7      | 8      | 9      | 0      | `      |
+ * |--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+-----------------|
+ * | TAB    | Q      | W      | E      | R      | T      | NUM7   | NUM8   | NUM9   | Y      | U      | I      | O      | P      | Å      |
+ * |--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+-----------------+--------|
+ * | LCTRL  | A      | S      | D      | F      | G      | NUM4   | NUM5   | NUM6   | H      | J      | K      | L      | ;      | Ä      |
+ * |--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------------------------+--------|
+ * | LSHIFT | Z      | X      | C      | V      | B      | NUM1   | NUM2   | NUM3   | N      | M      | <      | >      | UP     | -      |
+ * |--------+--------+--------+--------+--------+-----------------+--------+--------+--------+--------+-----------------+--------+--------|
+ * | MENU   | <      | LGUI   | LALT   | LOWER  | SPACE  | NUM00  | NUM0   | NUM.   | ENTER  | RAISE  | RALT   | LEFT   | DOWN   | RIGHT  |
+ * '--------------------------------------------------------------------------------------------------------------------------------------'
+ */
+							      
+ [_QWERTY] = { /* SPLIT QWERTY */
+  { KC_GESC, KC_1,    KC_2,    KC_3,    KC_4,    KC_5,    KC_PSCR, KC_SLCK, KC_LOCK, KC_6,   KC_7,    KC_8,    KC_9,    KC_0,    KC_MINS  },
+  { KC_TAB,  KC_Q,    KC_W,    KC_E,    KC_R,    KC_T,    KC_P7,   KC_P8,   KC_P9,   KC_Y,   KC_U,    KC_I,    KC_O,    KC_P,    KC_LBRC  },
+  { KC_LCTL, KC_A,    KC_S,    KC_D,    KC_F,    KC_G,    KC_P4,   KC_P5,   KC_P6,   KC_H,   KC_J,    KC_K,    KC_L,    KC_SCLN, KC_QUOT  },
+  { KC_LSFT, KC_Z,    KC_X,    KC_C,    KC_V,    KC_B,    KC_P1,   KC_P2,   KC_P3,   KC_N,   KC_M,    KC_COMM, KC_DOT,  MS_US,   KC_SLSH  },
+  { KC_APP,  KC_NUBS, KC_LGUI, KC_LALT, MS_LW,   KC_SPC,  MS_P00,  KC_P0,   KC_PDOT, KC_ENT, MS_RS,   KC_RALT, KC_LEFT, KC_DOWN, KC_RIGHT },
+ },
```

Comment:
Your keymap should be using a layout macro instead of the raw matrix assignments.

```suggestion
 [_QWERTY] = LAYOUT_ortho_5x15( /* SPLIT QWERTY */
   KC_GESC, KC_1,    KC_2,    KC_3,    KC_4,    KC_5,    KC_PSCR, KC_SLCK, KC_LOCK, KC_6,   KC_7,    KC_8,    KC_9,    KC_0,    KC_MINS,
   KC_TAB,  KC_Q,    KC_W,    KC_E,    KC_R,    KC_T,    KC_P7,   KC_P8,   KC_P9,   KC_Y,   KC_U,    KC_I,    KC_O,    KC_P,    KC_LBRC,
   KC_LCTL, KC_A,    KC_S,    KC_D,    KC_F,    KC_G,    KC_P4,   KC_P5,   KC_P6,   KC_H,   KC_J,    KC_K,    KC_L,    KC_SCLN, KC_QUOT,
   KC_LSFT, KC_Z,    KC_X,    KC_C,    KC_V,    KC_B,    KC_P1,   KC_P2,   KC_P3,   KC_N,   KC_M,    KC_COMM, KC_DOT,  MS_US,   KC_SLSH,
   KC_APP,  KC_NUBS, KC_LGUI, KC_LALT, MS_LW,   KC_SPC,  MS_P00,  KC_P0,   KC_PDOT, KC_ENT, MS_RS,   KC_RALT, KC_LEFT, KC_DOWN, KC_RGHT
 ),
```